src/Extract.hs

```diff
@@ -181,7 +185,12 @@ parse args = withGhc args $ \modules_ -> withTempOutputDir $ do
       _ <- setSessionDynFlags (GHC.ms_hspp_opts modsum)
       hsc_env <- getSession
 
-# if __GLASGOW_HASKELL__ >= 903
+# if __GLASGOW_HASKELL__ >= 904
+      hsc_env' <- liftIO (initializePlugins hsc_env)
```

Comment:
This code looks idential to the one in conditional for GHC >= 9.1 (`__GLASGOW_HASKELL__ >= 901`).
So, basically 9.2 and 9.4 behave the same.  Only 9.3 is different.  Is there a need to keep 9.3 (unreleased version)?

Also, wouldn't one make a minimal conditional (just 1 line is different --- and this only has an extra `Nothing`)?